mail/api.py

```diff
@@ -158,3 +169,37 @@ def send_financial_aid_email(cls, acting_user, financial_aid, subject, body):
                 email_body=body
             )
         return response
+
+
+def generate_financial_aid_email(financial_aid):
+    """
+    Generates the email subject and body for a FinancialAid status update. Accepted statuses are
+    FinancialAidStatus.APPROVED/FinancialAidStatus.REJECTED (the same email is sent for both statuses)
+    and FinancialAidStatus.PENDING_MANUAL_APPROVAL (documents have been received).
+    Args:
+        financial_aid (FinancialAid): The FinancialAid object in question
+    Returns:
+        dict: {"subject": (str), "body": (str)}
+    """
+    if financial_aid.status in [FinancialAidStatus.APPROVED, FinancialAidStatus.REJECTED]:
+        program_enrollment = ProgramEnrollment.objects.get(
+            user=financial_aid.user,
+            program=financial_aid.tier_program.program
+        )
+        message = FINANCIAL_AID_APPROVAL_MESSAGE.format(
+            program_name=financial_aid.tier_program.program.title,
+            price=get_formatted_course_price(program_enrollment)["course_price"]
+        )
+        subject = FINANCIAL_AID_APPROVAL_SUBJECT
```

Comment:
Can we change that to do the formatting here and in `FINANCIAL_AID_DOCUMENT_RECEIVED_SUBJECT`? It would be best to handle both cases explicitly, in case we change the template in the future and the parameters don't match up exactly anymore